torchci/lib/bot/mergeBot.ts

```diff
@@ -141,7 +141,7 @@ function mergeBot(app: Probot): void {
         // revert reason of 3+ words not given
         await addComment(
           ctx,
-          "Revert unsuccessful: please retry the command and provide a revert reason, " +
+          "Revert unsuccessful: please retry the command and provide at least 3 word long revert reasone, " +
```

Comment:
"reasone" -> "reason"

Reply:
Yeah, also running `yarn format` on the codebase